### PR TITLE
yet another agency ttl bug

### PR DESCRIPTION
**Pull request by kvahed**
http://jenkins.arangodb.biz:8080/job/arangodb-matrix-pr/6709/

### CODE REVIEW

File: arangod/Agency/State.cpp [hunk 1]
```diff
@@ -75,14 +75,16 @@ State::~State() = default;
 
 inline static std::string timestamp(uint64_t m) {
 
+  //TRI_ASSERT(m != 0);
+
   using namespace std::chrono;
-  
-  std::time_t t = (m == 0) ? std::time(nullptr) :
+
+  std::time_t t =
```
[the remark this answers is not in the document]
Comment (neunhoef) [reply arangod/Agency/State.cpp]: Maybe add a comment to explain.

**Comment by maierlars** [comment]
http://jenkins.arangodb.biz:8080/job/arangodb-matrix-pr/6730/

**Comment by kvahed** [comment]
https://jenkins01.arangodb.biz/view/PR/job/arangodb-matrix-pr/7017/

**Comment by kvahed** [comment]
image

**Pull request merged**